h2o-automl/src/test/java/ai/h2o/automl/targetencoding/TargetEncodingFrameHelperTest.java

```diff
@@ -215,4 +221,76 @@ public void registerTest() {
       Scope.exit();
     }
   }
+
+  private Map<String, Frame> getTEMapForTitanicDataset(boolean withFoldColumn) {
+    String foldColumnNameForTE = "te_fold_column";
+
+    Frame trainFrame = null;
+    try {
+      trainFrame = parse_test_file("./smalldata/gbm_test/titanic.csv");
+      String responseColumnName = "survived";
+      asFactor(trainFrame, responseColumnName);
+
+      if(withFoldColumn) {
+        int nfolds = 5;
+        addKFoldColumn(trainFrame, foldColumnNameForTE, nfolds, 1234);
+      }
+
+      BlendingParams params = new BlendingParams(3, 1);
+      String[] teColumns = {"home.dest", "embarked"};
+      TargetEncoder targetEncoder = new TargetEncoder(teColumns, params);
+      Map<String, Frame> testEncodingMap = targetEncoder.prepareEncodingMap(trainFrame, responseColumnName, withFoldColumn ? foldColumnNameForTE: null);
+      return testEncodingMap;
+    } finally {
+      if(trainFrame != null) trainFrame.delete();
+    }
+  }
+
+  // Checking that dfork is faster. This is a proof that dfork in this particular scenario is faster.
+  @Ignore()
+  @Test public void conversion_of_frame_into_table_doAll_vs_dfork_performance_test() {
```

Comment:
plase use camelCase for java method names

Reply:
@honzasterba I know the Rules but any rules should be intersected with rules from particular teams and projects and also should be questionable at any time and depend on the circumstances. Originally it was @seb-h2o's idea that it improves readability when it comes to long test names (see AutoMLTest.java). I tried to name it this way and personally agreed that it is easier to read. And for complex tasks when there are too many dimensions which should be tested you need to reflect particular setup in a test's name.